demo/glfw_opengl3/Makefile

```diff
@@ -2,7 +2,7 @@
 BIN = demo
 
 # Flags
-CFLAGS = -std=c99 -pedantic -O2
+CFLAGS ?= -std=c99 -pedantic -O2
```

Comment:
Well, in this case I would probably prefer `CFLAGS += -std=c99 -pedantic -O2`. What do you think?

Reply:
Changed. Also all other makefiles are changed now.